lib/actions/fileActions.js

```diff
@@ -207,27 +205,31 @@ export const updateFileAppRegions = () => (dispatch, getState) => {
         appStartAddress !== undefined &&
         appEndAddress !== undefined
     ) {
+        let restFileRegions = [];
```

Comment:
Maybe this could be written more like
```
const restFileRegions = fileRegions.filter(r => !( same lacomparision here));
```
It would be easier to see what is happening for my part at least, but there might have been something I've missed here.

Reply:
Yes. We have quite an amount of code, where “more modern” JS code should be used. Let us work toward that. 

In some cases it is a bit tricky, e.g. in the same state slice there is also`detectedRegionNames` which is a `Set`. That does not have a `filter` method but if we wanted to do a similar thing with that, then we could first convert that into an array and then call filter: `[...detectedRegionNames].filter(…)`.